examples/newPlacesNearbySearch/example.js

```diff
@@ -0,0 +1,133 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+// This example showcases the New Place nearbySearch API, as well as drawing a circle on a Google Map.
+// The example draws a circle on the map, for which the radius can be scaled dynamically.
+// The user can choose from different place type categories, and then perform a nearbySearch,
+// which will return any places matching the specified type within the circle radius.
+// This example also makes use of dynamic imports and advanced markers support.
+//
+// This is meant to be showcased as the client logic that is able to remain untouched
+// and retain the same functionality when using the Amazon Location Migration SDK.
+
+let map;
+let service;
+let markers = [];
+let searchCircle;
+
+function initMap() {
+  const austinCoords = new google.maps.LatLng(30.268193, -97.7457518); // Austin, TX :)
+
+  map = new google.maps.Map(document.getElementById("map"), {
+    center: austinCoords,
+    zoom: 13,
+    mapId: "DEMO_MAP_ID",
+  });
+
+  service = new google.maps.places.PlacesService(map);
+
+  // Initialize the circle
+  searchCircle = new google.maps.Circle({
+    strokeColor: "#146eb4",
+    strokeOpacity: 0.8,
+    strokeWeight: 2,
+    fillColor: "#146eb4",
+    fillOpacity: 0.15,
+    map,
+    center: austinCoords,
+    radius: 1500,
+  });
+
+  // Add radius slider listener
+  const radiusSlider = document.getElementById("radius");
+  const radiusValue = document.getElementById("radius-value");
+
+  radiusSlider.addEventListener("input", () => {
```

Comment:
Do we have remove listener example code?

Reply:
No, good point! I've added a task to go and add a general events example that showcases adding/removing listeners, and also the "once" variants of the listeners.